src/main/jbake/content/hawkular-metrics/docs/user-guide/index.adoc

```diff
@@ -1679,6 +1679,8 @@ then needs the prefix to uniquely identify the correct metric.
 
 |===
 
+Metrics has some special support for stream-based alerting, see <<Metrics Group Triggers>> for the details.
```

Comment:
Originally it was here but I moved it because I felt it was too much information in this section, I just wanted it to focus on the general stream-based alerting and not block a top-down reader from getting to the query-based section.  Honestly, I could go either way, if you feel it is better to put it here I will move it back.